src/sentry/conf/server.py

```diff
@@ -432,6 +432,14 @@ def env(key, default="", type=None):
 
 SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
 SESSION_COOKIE_NAME = "sentrysid"
+
+# setting SESSION_COOKIE_SAMESITE to None below for now because
+# Django's default in 2.1 now `Lax`.
+# this breaks certain IDP flows where we need cookies sent to us on a redirected POST
+# request, and `Lax` doesnt permit this.
+# See here: https://docs.djangoproject.com/en/2.1/ref/settings/#session-cookie-samesite
+SESSION_COOKIE_SAMESITE = None
```

Comment:
Docs say you need the string 'None' not a python None.

Reply:
No, it's python `None`.

Reply:
Maybe you're thinking about `CSRF_COOKIE_SAMESITE` in Django 3.1: https://docs.djangoproject.com/en/3.2/releases/3.1/#csrf

Reply:
seems like session cookie samesite supports the same "`None`" in 3.1 too https://docs.djangoproject.com/en/3.2/releases/3.1/#django-contrib-sessions

Reply:
and looking at it this morning, it appears that Chrome's behavior when there is no SameSite is "Lax + some POSTs" which is why it works without a value set. more info here: https://web.dev/samesite-cookies-explained/#samesitelax-by-default. will follow up with options in the near future.